shared/teams/team/settings-tab/index.stories.tsx

```diff
@@ -10,6 +10,7 @@ const commonProps = {
   ignoreAccessRequests: true,
   isBigTeam: true,
   loadWelcomeMessage: Sb.action('loadWelcomeMessage'),
+  onEditWelcomeMessage: () => {},
```

Comment:
```suggestion
  onEditWelcomeMessage: Sb.action('editWelcomeMessage'),
```

This way when you click the button the storyboard gives you a nice indicator of what was called